.travis.yml

```diff
@@ -39,7 +39,7 @@ before_script:
 - mysql -e 'SET @@GLOBAL.wait_timeout=1800'
 script:
 - if [ "${coverage}" == "" ]; then composer test; fi
-- if [ "${coverage}" != "" ]; then composer run coverage --timeout=0; fi
+- if [ "${coverage}" != "" ]; then composer run coverage --timeout=2000; fi
```

Comment:
So timeout=0 was supposed to remove the timeout entirely. Maybe that stopped working?